nh_eobs_mobile/controllers/main.py

```diff
@@ -575,6 +575,16 @@ def get_patients(self, *args, **kw):
             cr, uid, patient_api.get_followed_patients(
                 cr, uid, []), context=context)
 
+        for patient in patients:
+            if patient.get('frequency'):
+                patient['frequency_string'] = '{:2d} hour(s) {:02d} min(s)'.format(
```

Comment:
```suggestion
                patient['frequency_string'] = self.minutes_to_hours_minutes_string(patient.get('frequency'))

    def minutes_to_hours_minutes_string(self, minutes):
        return '{:2d} hour(s) {:02d} min(s)'.format(*divmod(minutes, 60))
```

function name is up to you